README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
```

Comment:
Ensure that the installation command for Go Task utilizes an appropriate package manager for the target environment, e.g., `apt`, `yum`, or `choco`, to enhance cross-platform compatibility.